app/views/catalog/_home_text.html.erb

```diff
@@ -23,6 +23,10 @@
                                                     types: [NumismaticIssue],
                                                     bulk: [] } %>
 
+<%= render partial: 'catalog/work_types', locals: { label: 'Media Reserves',
+                                                    types: [MediaReserve],
+                                                    bulk: [] } %>
+
```

Comment:
I was expecting MediaReserve to be in the same group as ScannedResource, not in its own section.

Reply:
I added it like this because otherwise I couldn't find a way to add a separate label for the Media Reserves button. When I kept it as [ScannedResource] vs [MediaReserve] then the panel in the home page had a title Media Reserves and a type Scanned Resource.

Reply:
If this proves bothersome, in the meantime I think we can delete the button and figure that out in another issue.